pkg/kubelet/kubelet.go

```diff
@@ -2603,6 +2736,7 @@ func (kl *Kubelet) LatestLoopEntryTime() time.Time {
 	return val.(time.Time)
 }
 
+// PLEGHealthCheck returns whether the PLEG is healty.
```

Comment:
I think this is needed to ensure that kubelet satisfies `HostInterface` ... we dont see to have a check of that actually declared in the code, i.e. var _ HostInterface = &kubelet{}